lib/pluginmanager/main.rb

```diff
@@ -27,6 +28,7 @@ class Main < Clamp::Command
       subcommand "install", "Install a plugin", LogStash::PluginManager::Install
       subcommand "uninstall", "Uninstall a plugin", LogStash::PluginManager::Uninstall
       subcommand "update", "Install a plugin", LogStash::PluginManager::Update
+      subcommand "package", "Package installed plugins and dependencies locally", LogStash::PluginManager::Package
```

Comment:
not very explanatory: maybe something like "Package currently installed plugins into a self contained file useful for offline installation" ??